### ISSUE TITLE
Docker and Arch testing

### ISSUE THREAD

**Issue by petems** [jantman/puppet-archlinux-macbookretina#10]
I see there's a branch for Arch in Docker support, seems awesome! 👍 

I opened a ticket: https://tickets.puppetlabs.com/browse/BKR-1021

But from my understanding, there's no systemd or init command that I can run for the `docker_cmd`... do you know what I'd need to run to get the box working?

**Comment by jantman** [comment]
I'm pretty sure I started that branch but never got it working... but that was also when I had very little Docker experience.

So the issue is that most people try to follow the "one process per container" Docker approach... as of when I looked around for a current Arch image, that's the way it was. So I believe someone would need to roll a custom Arch docker image that has systemd in it, and is essentially set up to "look" like a full Arch install.
